src/utils/find-plugin.ts

```diff
@@ -1,19 +1,29 @@
 import { Plugin } from '../plugins'
 
+/**
+ * Support requiring both CommonJS `module.exports` and ES5 `export` modules
+ *
+ * @param path The path to `require`
+ */
+const requireInterop = (path: string): any => {
+  const obj = require(path)
+  return obj && obj.__esModule ? obj['default'] : obj
```

Comment:
Hum I'm not sure that's what we want: that would restrict to only export a default object.

Eg.
```js
export function parseFile() {}
export function parseWorkspace() {}
```
wouldn't work anymore.

I think this would be more compatible:
```suggestion
  return obj && obj.__esModule && obj['default'] ? obj['default'] : obj
```

Reply:
actually that's why the tests are failing